lib/logstash/filters/ruby.rb

```diff
@@ -43,7 +43,11 @@ def filter(event,&block)
       @codeblock.call(event,&block)
       filter_matched(event)
     rescue Exception => e
-      @logger.error("Ruby exception occurred: #{e}")
+      err_details = {}
+      err_details["exception"] = e
+      err_details["event"] = event
```

Comment:
in order to avoid dumping the whole event every time there's an error, I suggest not logging it.
we already tag the event with the `_rubyexception` tag, which you can then route to an error log file.